src/styles/variables.scss

```diff
@@ -0,0 +1,12 @@
+$days: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+$day-size: 100px;
+$day-border: 1px;
+$day-gap: 1px;
+$day-padding: 10px;
+$default-bg: #eee;
+$hover-bg: #ffbfcb;
+$border-color: #000;
+$day-transform-duration: 0.5s;
+$day-font-size: 30px;
+$day-font-family: Arial, sans-serif;
+$calendar-width: $day-size * 7 + $day-gap * 6;
```

Comment:
The calculation for `$calendar-width` does not include the padding and border widths. If the design requires that the calendar width includes these, you'll need to adjust the calculation to account for them.